client/src/app/app.component.html

```diff
@@ -1,5 +1,5 @@
 <!--The content below is only a placeholder and can be replaced.-->
-<div style="text-align:center">
+<!-- <div style="text-align:center">
```

Comment:
This Incomplete `<!--` is the cause of your error as you haven't closed this comment, remove this.